helmexec/context.go

```diff
@@ -0,0 +1,18 @@
+package helmexec
+
+type HelmContext struct {
```

Comment:
I wonder if we can safely merge this into `execer`. I have been long thinking that `execer` is the "context" and adding an another context "HelmContext" isn't a must? 🤔

Reply:
No can do. `execer`s lifecycle is not tied to the release, but is allocated once per helmfile. And it is used in // per multiple threads. That is why I ended up passing a context to most methods.

Reply:
Ah that's good point. I dreamed of changing it to use a dedicated execer per release, but turned out it affects a lot of code including tests. Let's use HelmContext for now.

Thanks!